### ISSUE TITLE
TabError: inconsistent use of tabs and spaces in indentation

### ISSUE THREAD

**Issue by Schulzjo** [heartexlabs/label-studio#931]
**Describe the bug**
using docker-compose up ends in `label-studio_app_1 exited with code 1`

**To Reproduce**

1. Run 'git clone https://github.com/heartexlabs/label-studio.git'
2. Run 'docker-compose up'
3. See error

**Expected behavior**
Normal running Container

**Screenshots**
```
app_1    | [2021-05-18 12:10:01,133] [root::_read_py::52] [WARNING] Can't read version file: version_.py
app_1    | [2021-05-18 12:10:01,133] [root::_read_py::53] [WARNING] No module named 'version_'
app_1    | [2021-05-18 12:10:01,371] [asyncio::__init__::59] [DEBUG] Using selector: EpollSelector
app_1    | Traceback (most recent call last):
app_1    |   File "label_studio/manage.py", line 22, in <module>
app_1    |     execute_from_command_line(sys.argv)
app_1    |   File "/usr/local/lib/python3.8/dist-packages/django/core/management/__init__.py", line 401, in execute_from_command_line
app_1    |     utility.execute()
app_1    |   File "/usr/local/lib/python3.8/dist-packages/django/core/management/__init__.py", line 395, in execute
app_1    |     self.fetch_command(subcommand).run_from_argv(self.argv)
app_1    |   File "/usr/local/lib/python3.8/dist-packages/django/core/management/base.py", line 330, in run_from_argv
app_1    |     self.execute(*args, **cmd_options)
app_1    |   File "/usr/local/lib/python3.8/dist-packages/django/core/management/base.py", line 371, in execute
app_1    |     output = self.handle(*args, **options)
app_1    |   File "/usr/local/lib/python3.8/dist-packages/django/core/management/base.py", line 85, in wrapped
app_1    |     res = handle_func(*args, **kwargs)
app_1    |   File "/usr/local/lib/python3.8/dist-packages/django/core/management/commands/migrate.py", line 75, in handle
app_1    |     self.check(databases=[database])
app_1    |   File "/usr/local/lib/python3.8/dist-packages/django/core/management/base.py", line 392, in check
app_1    |     all_issues = checks.run_checks(
app_1    |   File "/usr/local/lib/python3.8/dist-packages/django/core/checks/registry.py", line 70, in run_checks
app_1    |     new_errors = check(app_configs=app_configs, databases=databases)
app_1    |   File "/usr/local/lib/python3.8/dist-packages/django/core/checks/urls.py", line 13, in check_url_config
app_1    |     return check_resolver(resolver)
app_1    |   File "/usr/local/lib/python3.8/dist-packages/django/core/checks/urls.py", line 23, in check_resolver
app_1    |     return check_method()
app_1    |   File "/usr/local/lib/python3.8/dist-packages/django/urls/resolvers.py", line 408, in check
app_1    |     for pattern in self.url_patterns:
app_1    |   File "/usr/local/lib/python3.8/dist-packages/django/utils/functional.py", line 48, in __get__
app_1    |     res = instance.__dict__[self.name] = self.func(instance)
app_1    |   File "/usr/local/lib/python3.8/dist-packages/django/urls/resolvers.py", line 589, in url_patterns
app_1    |     patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
app_1    |   File "/usr/local/lib/python3.8/dist-packages/django/utils/functional.py", line 48, in __get__
app_1    |     res = instance.__dict__[self.name] = self.func(instance)
app_1    |   File "/usr/local/lib/python3.8/dist-packages/django/urls/resolvers.py", line 582, in urlconf_module
app_1    |     return import_module(self.urlconf_name)
app_1    |   File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
app_1    |     return _bootstrap._gcd_import(name[level:], package, level)
app_1    |   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
app_1    |   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
app_1    |   File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
app_1    |   File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
app_1    |   File "<frozen importlib._bootstrap_external>", line 783, in exec_module
app_1    |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
app_1    |   File "/label-studio/label_studio/core/urls.py", line 58, in <module>
app_1    |     re_path(r'^', include('data_manager.urls')),
app_1    |   File "/usr/local/lib/python3.8/dist-packages/django/urls/conf.py", line 34, in include
app_1    |     urlconf_module = import_module(urlconf_module)
app_1    |   File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
app_1    |     return _bootstrap._gcd_import(name[level:], package, level)
app_1    |   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
app_1    |   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
app_1    |   File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
app_1    |   File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
app_1    |   File "<frozen importlib._bootstrap_external>", line 783, in exec_module
app_1    |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
app_1    |   File "/label-studio/label_studio/data_manager/urls.py", line 6, in <module>
app_1    |     from data_manager import api, views
app_1    |   File "/label-studio/label_studio/data_manager/api.py", line 212
app_1    |     return DataManagerTaskSerializer
app_1    |                                    ^
app_1    | TabError: inconsistent use of tabs and spaces in indentation
label-studio_app_1 exited with code 1
```

**Environment (please complete the following information):**
 - OS: Ubuntu 20.04

**Additional context**
Yesterday I started to inspect label-studio but run in similar error to #942.
Today I pulled again actual master and tried to rebuild. Then I got the TabError above. I tried to reformat code in api.py file but when I rebuild the container with `docker-compose build --no-cache` my code changes get not adopted. Do I have to change something on dockerfiles to use my local codebase?

**Comment by makseq** [comment]
@Schulzjo Could you take the latest commit from the master branch?

**Comment by Schulzjo** [comment]
still the same error.

I did:

`git pull`
`docker-compose build --no-cache`
`docker-compose up`

But I am still confused about the docker behavior. My container seems not to use actual codebase.
For test reasons I added a` dummie_class `above the` get_serializer_class` in `api.py` which produce the error. And the error is still in line 212, but should be now in 214, or do I misunderstand something?

I am a docker newbie and can not identify why the container do not get the actual code changes.
Anny suggestions?

Thx
